pycytominer/cyto_utils/cells.py

```diff
@@ -429,13 +429,17 @@ def get_sql_table_col_names(self, table):
 
         return meta_cols, feat_cols
 
-    def load_compartment(self, compartment):
+    def load_compartment(self, compartment, float_datatype: type = np.float64):
         """Creates the compartment dataframe.
 
         Parameters
         ----------
         compartment : str
             The compartment to process.
+        float_datatype: type, default np.float64
+            Numpy floating point datatype to use for load_compartment and resulting
+            dataframes. Please note: using any besides np.float64 are experimentally
```

Comment:
It might be worth noting other options one might use in this docstring and including what the anticipated benefit might be. (i.e. - why is this worth adding as an argument and why anyone would ever change it?)

Reply:
Great point, thank you - I'll update this to include a mention about the relationship to performance and an example.